libraries/lib-wave-track/WaveClip.h

```diff
@@ -326,6 +338,43 @@ class WAVE_TRACK_API WaveClip final :
       */
    );
 
+   /*!
+    * @param t relative to clip start sample
+    */
+   bool
+   GetFloatAtTime(double t, size_t iChannel, float& value, bool mayThrow) const;
+
+   //! Succeed with out-of-bounds requests, only changing what is in bounds.
+   //! @{
+   // clan-format off
```

Comment:
You mean `clang-format` here and again below

Reply:
No, I meant `clan-format` XD